.github/pipeline.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What does this file do?

Reply:
It's a VSCode project file. It's just a shortcut to open the project. It can do multiple things, like putting together folders from different paths etc. Anyway, now atm this file just opens the folder where it's place.